### PR TITLE
Fixed 'alt text was rendering with inline image in pdf' issue

### PR DESCRIPTION
**Pull request by anuradhas248**
Fixed 'alt text was rendering with inline image in pdf' issue and also allowed http with https for image url.

Jira ID - KMS-4389
Git Issue - [505](https://github.com/opensalt/opensalt/issues/505)

---
This change is [Reviewable](https://reviewable.io/reviews/opensalt/opensalt/506)

### CODE REVIEW

File: src/Controller/PdfExportController.php [hunk 1]
```diff
@@ -69,8 +69,8 @@ function () use ($phpWordObject) {
      */
     public function render_images($string)
     {
-        $pattern = '/\!\[([^\]]*)\]\(((?:https:\/\/|\/)[^\)]+)\)/';
-        $result = preg_replace($pattern, '$1 <img src = "$2">', $string);
+        $pattern = '/\!\[([^\]]*)\]\(((?:(?:https?|):\/\/|\/)[^\)]+)\)/';
```
Comment (roverwolf) [inline src/Controller/PdfExportController.php]: What was the intention of this change?  Do we really want to allow `://something-here` to be valid?  Or was the intention to just add the `?` after the `s` to allow `http`?

Comment (roverwolf) [reply src/Controller/PdfExportController.php]: Or was the pattern supposed to change at all?  Just the replacement (removing $1)?

Comment (anuradhas248) [reply src/Controller/PdfExportController.php]: Testers have found few urls that used pattern alt text. And http was not allowing in previous pattern. So for allow http and pattern similar like "://something-here." I updated pattern. At the same time I removed $1 as it was showing first bracket text along with image.

**Comment by anuradhas248** [comment]
@roverwolf - As per your suggestions, I have updated pattern that allowing http(as some of content used http url to render image) along with https. And removed second condition which was allowing :// prefix.
Please verify.
Thank you.

**Pull request merged**